release_tester/arangodb/installers/tar.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+""" run an installer for the debian based operating system """
+import sys
+import time
+import os
+import shutil
+import logging
+from pathlib import Path
+import psutil
+from arangodb.instance import ArangodInstance
+from arangodb.installers.base import InstallerBase
+from tools.asciiprint import ascii_print
+logging.basicConfig(level=logging.INFO, format='%(asctime)s %(message)s')
+import tools.loghelper as lh
+import semver
+
+
+class InstallerTAR(InstallerBase):
+    """ install .deb's on debian or ubuntu hosts """
+    def __init__(self, cfg):
+
+        self.tar = 'tar'
+        self.check_stripped = True
+        self.check_symlink = True
+        self.server_package = None
+        self.client_package = None
+        self.debug_package = None
+        self.log_examiner = None
+        version = cfg.version.split("~")[0]
+        version = ".".join(version.split(".")[:3])
+        self.semver = semver.VersionInfo.parse(version)
+
+        cfg.have_system_service = False
+        cfg.baseTestDir = Path('/tmp')
+        cfg.installPrefix = None
+        cfg.bin_dir = None
+        cfg.sbin_dir = None
+        cfg.real_bin_dir = None
+        cfg.real_sbin_dir = None
+        cfg.localhost = 'ip6-localhost'
+
+        cfg.logDir = None
+        cfg.dbdir = None
+        cfg.appdir = None
+        cfg.cfgdir = None
+
+
+        super().__init__(cfg)
+
+    def calculate_package_names(self):  
+        # Are those required to be stored in the cfg?
+        enterprise = 'e' if self.cfg.enterprise else ''
+        # package_version = '1'
+        architecture = 'linux'
```

Comment:
add 
```
    macver = platform.mac_ver()
    if macver[0]:
```
and correctly evaluate the filename of the macos tar; the `linux` one needs to become the `else` case.